### PR TITLE
 Added support for mailto: URI on NFC tag

### LINKED ISSUE #7755: Add support for mailto: URI on NFC tag
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.801

### Where did you get the app from?

Google Play

### Android version

12 SKQ1.210908.001

### Device model

Xiaomi Redmi Note 10 Pro

### Steps to reproduce

1. Get a physical RFID tag (e.g. NTAG213 or NTAG215 ...)
2. Use the "NFC Tools" or "NFC Tools PRO" App to write some records like text, a phone-number,  **and an E-Mail-Adress ** on the tag. (The app automatically converts this to a list of URLs like phone: or mailto:
3. hold the tag to a NFC-capable android phone with K-9 installed and configured with an account

### Expected behavior

As soon as the phone (start screen, no app running, but NFC enabled in settings) gets contact to the tag, it beeps and presents a list of apps to launch, for each of the URLs the apps ready to process the given URL. 

For e-mail-addresses, K-9 should appear in the list of apps. 

### Actual behavior

K-9 does not appear on the list of apps, while other do. Gmail appears. Just for testing, without ever opening or configuring them, other mail apps like Blue Mail or AquaMail get listed as well. Just not K-9. See the screenshot

60d0ee52e6740112d8b513b55a308a4c97df244c
 
Interestingly, opening a URL like mailto:abc@danisch.de through the chrome-browser works, K-9 is offered as a choice. 

### Logs

60d0ee52e6740112d8b513b55a308a4c97df244c

### PR DESCRIPTION
**Pull request by Silas217209**
Closes #7755

### CODE REVIEW

File: app/common/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -143,6 +143,11 @@
                 <data android:scheme="mailto" />
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.nfc.action.NDEF_DISCOVERE" />
```
Comment (cketti) [inline app/common/src/main/AndroidManifest.xml:147]: The action name is missing a "D" at the end.

```suggestion
                <action android:name="android.nfc.action.NDEF_DISCOVERED" />
```

**Review by cketti — CHANGES_REQUESTED** [review]
Thanks for the pull request :+1:

I've tested this with the action name fixed and it works as intended. So once you fix the manifest entry, this should be good to go.

File: app/common/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Silas217209) [reply app/common/src/main/AndroidManifest.xml]: That was a dumb mistake. Fixed it!

**Review by cketti — APPROVED** [review]
:+1:

**Comment by cketti** [comment]
Thank you!

**Pull request merged**